katas/content/distinguishing_unitaries/h_x/Solution.qs

```diff
@@ -0,0 +1,12 @@
+namespace Kata {
+    open Microsoft.Quantum.Measurement;
```

Comment:
This namespace is open by default as part of the prelude, so we don't need to open it
```suggestion
```